### PR TITLE
Mailer

### PR DESCRIPTION
**Pull request by ntsanq**

### CODE REVIEW

File: src/Controller/Mail/MailController.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+     * @throws Exception
+     */
+    #[Route('/mail', name: 'index', methods: 'POST')]
+    public function index(MailService $mailService, Request $request)
+    {
+        $bodyRequest = json_decode($request->getContent(), true);
+        $userMail = $bodyRequest['userMail'];
+        $name = $bodyRequest['name'];
+        $mailService->mail($userMail, self::FILE, $name);
+        return $this->success([
```
Comment (Ductuan92) [inline src/Controller/Mail/MailController.php]: this should have a new line in between

File: src/Mapper/TicketArrayToTicket.php [hunk 2]
```diff
@@ -31,6 +31,7 @@ public function __construct(
     public function mapper($metadata)
     {
         $ticket = new Ticket();
+        var_dump($metadata);
```
Comment (Ductuan92) [inline src/Mapper/TicketArrayToTicket.php]: this is unacceptable LOL

File: src/Service/MailService.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+            $mail->setFrom($this->containerBag->get('zohoMail'), self::SUPPORT_MAIL_NAME);
+            $mail->addAddress($userMail, $userName);
+
+            $mail->isHTML(true);
+            $mail->Subject = "Hello $userName";
+            $mail->Body = file_get_contents($path);
+            $mail->send();
+        } catch (Exception $e) {
+            throw new \Exception("Message could not be sent. Mailer Error: {$mail->ErrorInfo}");
```
Comment (Ductuan92) [inline src/Service/MailService.php:41]: This should use 'use Exception' instead

**Review by Ductuan92 — CHANGES_REQUESTED** [review]

File: src/Mapper/TicketArrayToTicket.php (hunk 2 above)
Comment (ntsanq) [reply src/Mapper/TicketArrayToTicket.php]: aiii chhaaaaaa, forgive me

**Review by tolehoai — APPROVED** [review]
I approve with this

**Pull request merged**